dependency_support/org_theopenroadproject_asap7_pdk_r1p7/asap7.bzl

```diff
@@ -78,5 +82,10 @@ asap7_cell_library = rule(
             executable = True,
             cfg = "exec",
         ),
+        "_uncompress": attr.label(
+            default = Label("@7zip//:7za"),
```

Comment:
Do we have this tool imported inside Google?

Reply:
Looks like it.